src/components/FormSchemaEditor/Select.tsx

```diff
@@ -0,0 +1,100 @@
+import { Listbox, Transition } from "@headlessui/react";
+import _ from "lodash";
+import { Fragment } from "react";
+
+interface UpDownArrowProps {}
+const UpDownArrow: React.FC<UpDownArrowProps> = () => {
```

Comment:
On a previous commit of mine, @kevinl120 commented about the downsides of React.FC- I don't know if it also applies here. [reference](https://github.com/facebook/create-react-app/pull/8177) and [reference2](https://www.harrymt.com/blog/2020/05/20/react-typescript-react-fc.html)